docs/en/latest/FAQ.md

```diff
@@ -79,7 +79,7 @@ Ref to FAQ #7.
 
 ### 9. Modify the Admin API key in APISIX-Ingress
 
-Usually, you need to modify the Admin API key to protect Apache APISIX. Please refer to this [link](https://apisix.apache.org/docs/apisix/how-to-build/#updating-admin-api-key) to simply change Apache APISIX.
+Usually, you need to modify the Admin API key to protect Apache APISIX. Please refer to this [link](https://apisix.apache.org/docs/apisix/admin-api/) to simply change Apache APISIX.
```

Comment:
The correct link is 
https://apisix.apache.org/docs/apisix/installation-guide/#updating-admin-api-key

Reply:
I felt sorry :(, and push a new commit